iTaylorCodes/Gamr
# Setup User Signup Page

### What does this PR do?
- This Pull Request sets up the user signup feature
- This PR closes #18 
### Description of Task to be completed
- Setup /signup view. Setup error handling. Setup users/signup template. Add styles to signup page. Create tests for signup view and error handler.
### How should this be manually tested?
- Visit /signup, fill out signup form and confirm submission of new user into database.
### Video Example
https://user-images.githubusercontent.com/76837421/130182969-dcd60cf7-b0d6-48ae-9bf4-e1b33f0322e5.mov
